taichi/transforms/ir_printer.cpp

```diff
@@ -202,6 +206,14 @@ class IRPrinter : public IRVisitor {
     print("while control {}, {}", stmt->mask->name(), stmt->cond->name());
   }
 
+  void visit(ContinueStmt *stmt) override {
+    if (stmt->scope) {
+      print("{} continue (as_return={})", stmt->name(), stmt->as_return());
```

Comment:
This `(as_return={})` seems unnecessary since we always know it's as_return by seeing it's in the top-level.

Reply:
That's true, but for debugging the IR we may want to display as much information about the statements as possible.

Reply:
Would displaying `stmt->scope->name()` more useful? On the other hand, you can also just eyeball to figure out which loop the continue acts on..

Reply:
> Would displaying `stmt->scope->name()` more useful?

I think that would be useful. In general, it will be a good idea to ensure we can easily reconstruct the IRNodes from the printed IR string. We might need to implement an IR parser in the future.

Reply:
OK, fixed.